src/vs/workbench/contrib/preferences/browser/settingsTreeModels.ts

```diff
@@ -193,6 +193,8 @@ export class SettingsTreeSettingElement extends SettingsTreeElement {
 			this.valueType = SettingValueType.Number;
 		} else if (this.setting.type === 'boolean') {
 			this.valueType = SettingValueType.Boolean;
+		} else if (this.setting.type === 'array' && this.setting.listItemType === 'string') {
+			this.valueType = SettingValueType.Array;
```

Comment:
This could still be ArrayOfString or whatever, I was just pointing out 'list' vs 'array'. But it's up to you